src/transformers/models/electra/modeling_tf_electra.py

```diff
@@ -140,8 +140,8 @@ def call(
         elif past_key_value is not None:
             key_layer = self.transpose_for_scores(self.key(inputs=hidden_states), batch_size)
             value_layer = self.transpose_for_scores(self.value(inputs=hidden_states), batch_size)
-            key_layer = tf.concatenate([past_key_value[0], key_layer], dim=2)
-            value_layer = tf.concatenate([past_key_value[1], value_layer], dim=2)
+            key_layer = tf.concat([past_key_value[0], key_layer], axis=2)
+            value_layer = tf.concat([past_key_value[1], value_layer], axis=2)
```

Comment:
What about the attention_mask here? I.e.: https://github.com/huggingface/transformers/pull/15774/files#r813750038 - don't we need to change that here as well?

Reply:
This block was copied from `BertSelfAttention`

https://github.com/huggingface/transformers/blob/d1fcc90abf34cc498c8a65a717ad0d9354ceca97/src/transformers/models/bert/modeling_bert.py#L275

But let me check the details, and probably add some remarks to avoid confusion in the future (or fix it if it is an error - but I think unlikely)

Reply:
The query contains only the **present** inputs I think.

Reply:
sounds good!